target/linux/lantiq/image/vr9.mk

```diff
@@ -118,6 +118,18 @@ define Device/avm_fritz3370-rev2-micron
 endef
 TARGET_DEVICES += avm_fritz3370-rev2-micron
 
+define Device/avm_fritz3390
+  $(Device/AVM)
+  $(Device/NAND)
+  DEVICE_MODEL := FRITZ!Box 3390
+  SOC := vr9
```

Comment:
SOC:=9 is unnececary. Default SoC is already set in openwrt/target/linux/lantiq/image/Makefile.